Task.js

```diff
@@ -127,6 +133,11 @@ class Task extends Unit {
       exp_short_name: lab_data.exp_short_name,
     };
 
+	  if(this.lu === "")
+	  {
+		 page_data.meta.learning_unit = exp_info.name;
```

Comment:
Do we need this separate code to set learning_unit? Can we just not change line 118 to:
learning_unit: this.lu || exp_info.name